eva-lib/src/test/java/uk/ac/ebi/eva/lib/repository/VariantEntityRepositoryTest.java

```diff
@@ -418,21 +419,21 @@ public void testFindByRegionsAndComplexFiltersExcludeAttributes() {
 
     @Test
     public void testFindDistinctChromosomesByStudyId() {
-        List<String> chromosomeList = variantEntityRepository.findDistinctChromosomes();
+        Set<String> chromosomeSet = variantEntityRepository.findDistinctChromosomes();
 
-        List<String> expectedChromosomeList = new ArrayList<>();
-        expectedChromosomeList.add("11");
-        expectedChromosomeList.add("9");
-        expectedChromosomeList.add("2");
+        Set<String> expectedChromosomeSet = new HashSet<>();
```

Comment:
Maybe just `expectedChromosomes` as variable name? It doesn't really matter what the data structure is.